source/styles/style.less

```diff
@@ -190,16 +190,49 @@ span.disabled {
   position: absolute;
   right: 40px;
   top: 10px;
-  color: white;
```

Comment:
Out of interest, why the change here?

Reply:
Mainly for certainty. Just me being picky :)